config/ethernal.ts

```diff
@@ -7,8 +7,7 @@ export const getEthernalConfig = (
   const ethernal: HardhatUserConfig['ethernal'] =
     ETHERNAL &&
     typeof ETHERNAL_EMAIL === 'string' &&
-    typeof ETHERNAL_PASSWORD === 'string' &&
-    ['localhost', 'hardhat'].includes(hre.network.name)
```

Comment:
does ethernal work on prod networks?

Reply:
No but this blocks it working for a localhost standalone node because hre is not available at this stage of the config assembly.  I think relying on `ETHERNAL` being true is sufficient.